BookingsApi/BookingsApi.UnitTests/Validation/ParticipantRequestValidationTests.cs

```diff
@@ -131,7 +131,29 @@ private ParticipantRequest BuildRequest()
         {
             return Builder<ParticipantRequest>.CreateNew()
                  .With(x => x.CaseRoleName = "Applicant")
-                 .With(x => x.HearingRoleName = "Representative")
+                 .With(x => x.HearingRoleName = HearingRoleName.Representative)
+                 .Build();
+        }
+
+        [Test]
+        public async Task Should_return_missing_telephone_error()
+        {
+            var request = BuildStaffMemberRequest();
+            request.TelephoneNumber = string.Empty;
+
+            var result = await _validator.ValidateAsync(request);
+
+            result.IsValid.Should().BeFalse();
+            result.Errors.Count.Should().Be(1);
+            result.Errors.Any(x => x.ErrorMessage == ParticipantRequestValidation.NoTelephoneErrorMessage)
+                .Should().BeTrue();
+        }
+
+        private ParticipantRequest BuildStaffMemberRequest()
+        {
+            return Builder<ParticipantRequest>.CreateNew()
+                 .With(x => x.CaseRoleName = "Staff Member")
```

Comment:
Should this be HearingRoleName.StaffMember instead of the literal, as you have used in the next assignment?

Reply:
Hearing Role and Case Role is the same for the Staff Member but diff for other users. This could bring a bit of confusion if we use the same dictionary (HearingRoleName.StaffMember)